test/spec/zeebe/ZeebeVariableResolver.spec.js

```diff
@@ -633,6 +634,85 @@ describe('ZeebeVariableResolver', function() {
 
   });
 
+
+  describe('connectors', function() {
+    beforeEach(
+      bootstrapModeler(connectorsXML, {
+        container,
+        additionalModules: [
+          ZeebeVariableResolverModule
+        ],
+        moddleExtensions: {
+          zeebe: ZeebeModdle
+        }
+      })
+    );
+
+
+    it('should NOT add variables on missing headers', inject(async function(variableResolver, elementRegistry) {
+
+      // given
+      const task = elementRegistry.get('empty');
+
+      // when
+      const variables = await variableResolver.getVariablesForElement(task.businessObject);
+
+      // then
+      expect(variables).to.variableEqual([ ]);
+    }));
+
+
+    it('should add variables from resultVariable header', inject(async function(variableResolver, elementRegistry) {
+
+      // given
+      const task = elementRegistry.get('resultVariable');
+
+      // when
+      const variables = await variableResolver.getVariablesForElement(task.businessObject);
+
+      // then
+      expect(variables).to.variableEqual([
+        { name: 'resultVariable', origin: [ 'resultVariable' ] },
+      ]);
+    }));
+
+
+    it('should add variables from resultExpression header', inject(async function(variableResolver, elementRegistry) {
+
+      // given
+      const task = elementRegistry.get('resultExpression');
+
+      // when
+      const variables = await variableResolver.getVariablesForElement(task.businessObject);
+
+      // then
+      expect(variables).to.variableEqual([
+        { name: 'expressionVariable', origin: [ 'resultExpression' ] },
+        { name: 'anotherExpressionVariable', origin: [ 'resultExpression' ] },
+      ]);
+
+    }));
+
+
+    it('should add variables from both headers', inject(async function(variableResolver, elementRegistry) {
+
+      // given
+      const task = elementRegistry.get('both');
+
+      // when
+      const variables = await variableResolver.getVariablesForElement(task.businessObject);
+
+      // then
+      expect(variables).to.variableEqual([
+        { name: 'resultVariable', origin: [ 'both' ] },
```

Comment:
Both is the ID of the element, right?

Reply:
Yes, that's the element ID 😄

Reply:
A little confusing. :)

Reply:
I'll do some renameings on the things you mentioned before merging then 😄